fontes/avaliador-sintatico/avaliador-sintatico-base.ts

```diff
@@ -37,6 +37,12 @@ export abstract class AvaliadorSintaticoBase implements AvaliadorSintaticoInterf
     atual: number;
     blocos: number;
 
+    // Função util para birl onde o tipo int, short int e long int tem a mesma tratativa
+    consumirSemError(tipo: string): Boolean {
```

Comment:
Não acho necessário porque `verificarSeSimboloEIgualA()` já faz esse consumo de símbolo sem erros.

Reply:
```verificarSeSimboloEIgualA()``` realmente não retorna nenhum erro.
O problema é que a função ``this.consumir()`` estoura um erro caso a verificação seja false.
Isso me complicou na hora de adicionar alguns tipo meio inúteis que birl tem como ``MONSTRINHO`` e ``MONSTRAO`` que em ``C`` são ``short int`` e ``long int`` respectivamente mas para nossa regra de negocio não tem nenhuma serventia. Eu resolvi tratar os dois tipo como um ``int`` padrão nodejs assim como o tipo ``MONSTRO``. E pensei nessa forma de implementar, se tiver uma ideia melhor me informa pra corrigir.

Reply:
Então, basta usar `verificarSeSimboloEIgualA()` ao invés de usar essa `consumirSemError()`.

Reply:
Ok, farei correção.